docs/contribute/style_guide.rst

```diff
@@ -12,10 +12,13 @@ your repository:
 
     pre-commit install
 
-This adds a pre-commit hook that automatically formats your code and conducts style checks before each commit (if you
-are interested in the set of rules that we are using for formatting the code, take a look at the ``[tool.ruff]`` section
-in the :github:`pyproject.toml <tree/master/pyproject.toml>` and the :github:`.clang-format <tree/master/.clang-format>`
-file). For manual checks at any time, execute:
+This adds a pre-commit hook that automatically formats your code and conducts style checks before each commit. We
+decided to enable all available `ruff rules <https://docs.astral.sh/ruff/rules/>`_ and disable only specific rules that
```

Comment:
I would keep it shorter: "The pre-commit hook applies most `ruff rules <https://docs.astral.sh/ruff/rules/>`_ (see the ``[tool.ruff]`` section in the :github:`pyproject.toml <tree/master/pyproject.toml>` file) and some C++ clang rules (see the :github:`.clang-format <tree/master/.clang-format>` file).